app/main.ipynb

```diff
@@ -275,17 +303,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
-   "outputs": [],
-   "source": [
-    "# write your code for task 9 here"
-   ],
+   "source": "data_set.corr(numeric_only=True)",
```

Comment:
The correlation analysis is performed, but there is no interpretation or highlighting of the connections between fields. Consider adding a step to interpret and highlight significant correlations.